### DISCUSSION TITLE
Support plans for Composer Patches v1

### DISCUSSION THREAD

**Discussion by mxr576** [cweagans/composer-patches#480] [General]
Now that [Composer Patches 2.0.0 on the horizion](https://www.cweagans.net/2023/02/seeking-feedback-on-the-dev-version-of-composer-patches/). I think I am not the only one who is curious about the support plans for Composer Patches 1.x. How urgently we need to update to Composer Patches 2.x?

**Comment by cweagans** [comment]
I haven't supported 1.x very well prior to 2.x being close, so I'm not sure it's urgent. It won't stop working when 2.0.0 is released.

That said, if you run into a problem on 1.x, I probably won't be eager to troubleshoot/fix if the issue has been fixed in 2.x. I'd really like to see 2.x adopted quickly, but there are reasons why you wouldn't upgrade right away (if you need the dependency patch resolution functionality, for instance).
